drivers/video/ili9341.c

```diff
@@ -154,7 +165,17 @@ dev_err(dev, "%s() :%d\n", __func__, __LINE__);
 			    ILI9341_MADCTL_MX;
 		break;
 	}
-	addr_mode |= ILI9341_MADCTL_BGR;
+
+	if (priv->is_st7789s_variant = ili9341_is_st7789s_variant(dev)) {
+		printf("Display Variant: st7789s\n");
+		env_set("display_chip", "st7789s");
+		addr_mode = 0xC0;
```

Comment:
@zonque This is what I would use to enable either of the DT nodes for the kernel later on...